index.html

```diff
@@ -6604,9 +6632,12 @@ <h3 id="Elemental">Elemental</h3>
                </td>
                <td>Cyclone Leap
                </td>
-               <td>Choose a point you can see within 25ft and jump to it. All creatures
-                  within 15ft of where you land are pulled 10ft closer to you if possible. 1 FORT
-                  point to resist the pull.
+               <td>
+                  <strong>Jump with the wind and pull your foes into striking distance.</strong>
+                  <p>
+                     Select a point within 25ft and jump to it. Then all creatures
+                     within 15ft of that point are pulled 10ft closer to you if possible. 1 FORT
```

Comment:
"are pulled up to 10ft closer".  10ft closer if possible makes it sound like a binary decision, they'll never be pulled 5ft

Reply:
done